water.mss

```diff
@@ -52,19 +51,37 @@
       }
     }
   }
-
-  [natural = 'mud'][zoom >= 10]::natural {
-    polygon-fill: @mud;
-    polygon-pattern-file: url('symbols/mud.png');
-    polygon-pattern-alignment: global;
-  }
 }
 
 #water-areas-overlay {
-  [natural = 'marsh'],
-  [natural = 'wetland'] {
-    [zoom >= 10] {
-      polygon-pattern-file: url('symbols/wetland.png');
+  [zoom >= 10] {
+    polygon-pattern-file: url('symbols/wetland.png');
+    polygon-pattern-alignment: global;
+  }
+  [zoom >= 14] {
+    [int_wetland = 'marsh'],
+    [int_wetland = 'saltmarsh'],
+    [int_wetland = 'wet_meadow'],
+    [int_wetland = 'fen'] {
```

Comment:
Would be fine with me - i included it since it is on http://wiki.openstreetmap.org/wiki/Key:wetland.